components/DatasetsTable/filters/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './FileTypeFilter';
+export * from './LabelFilter';
```

Comment:
Just for convention, I've typically used an index file when a folder corresponds to *one* main component and I then re-export that in the index.

Reply:
So basically I just remove the index file and reimport the components? Other than that, is the file structure corresponding to the rest of the project?